docs/getting-started.md

```diff
@@ -37,28 +37,13 @@ The generic requirements are explained below:
      If you need help setting up access to `/dev/kvm`, you should check out
      [Appendix A](#appendix-a-setting-up-kvm-access).
 
-<details>
-
-<summary>Click here to see a BASH script that will check if your system meets
+<summary>Run the command below to check if your system meets
 the basic requirements to run Firecracker.</summary>
 
 ```bash
-err="";
-[ "$(uname) $(uname -m)" = "Linux x86_64" ]  \
-	|| [ "$(uname) $(uname -m)" = "Linux aarch64" ] \
-	|| err="ERROR: your system is not Linux x86_64 or Linux aarch64."; \
-	[ -r /dev/kvm ] && [ -w /dev/kvm ] \
-	|| err="$err\nERROR: /dev/kvm is innaccessible."; \
-	[ $(( $(uname -r | cut -d. -f1)*1000 + $(uname -r | cut -d. -f2) >= 4014 )) -eq 1 ] \
-	|| err="$err\nERROR: your kernel version ($(uname -r)) is too old."; \
-	dmesg | grep -i "hypervisor detected" \
-	&& echo "WARNING: you are running in a virtual machine." \
-	&& echo "Firecracker is not well tested under nested virtualization."; \
```

Comment:
nit-ish: this check is not present in `tools/devtool`, while it's not critical it is nice to have.

Reply:
I've added it in the last commit and squashed the documentation commits but commited the `tools/devtool` change independently.